hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -6525,6 +6525,21 @@
       If the namespace is DEFAULT, it's best to change this conf to other value.
     </description>
   </property>
+  <property>
+    <name>dfs.client.congestion.backoff.mean.time</name>
+    <value>5000</value>
+    <description>
+      The mean milliseconds which is used to compute client congestion backoff sleep time.
```

Comment:
The mean time in milliseconds which is used to compute client congestion backoff sleep time.